package/MDAnalysis/core/topologyattrs.py

```diff
@@ -416,8 +457,16 @@ def get_segments(self, sg):
 
 #TODO: update docs to property doc
 class Resids(ResidueAttr):
+    """Residue ID"""
     attrname = 'resids'
     singular = 'resid'
+    transplants = defaultdict(list)
+
+    def atomget(atom):
+        return atom.residue.resid
+
+    transplants['atom'].append(
+        ('resid', property(atomget, None, None, atomget.__doc__)))
```

Comment:
I think the many to one mappings you list are clear though.  Plus it's pretty intuitive because of how pdbs and gros are set out. 

This link is read-only, so it shouldn't break our model.

Plus it's how it used to work, so it's one less API break?